network_importer/adapters/netbox_api/adapter.py

```diff
@@ -256,14 +256,18 @@ def convert_interface_from_netbox(
 
         if site and intf.tagged_vlans and import_vlans:
             for vid in [v.vid for v in intf.tagged_vlans]:
-                vlan, _ = self.get_or_create_vlan(vlan=self.vlan(vid=vid, site_name=site.name), site=site)
-                interface.allowed_vlans.append(vlan.get_unique_id())
+                try:
+                    vlan = self.get(self.vlan, identifier=dict(vid=vid, site_name=site.name))
+                    interface.allowed_vlans.append(vlan.get_unique_id())
+                except ObjectNotFound:
+                    LOGGER.debug("%s | VLAN %s is not present for site %s", self.name, vid, site.name)
```

Comment:
plese can you make that a WARNING message instead of DEBUG

Reply:
Updated.